R/cfbd_pbp_data.R

```diff
@@ -2038,7 +2038,7 @@ prep_epa_df_after <- function(dat) {
 clean_drive_info <- function(drive_df) {
   clean_drive <- drive_df %>%
     dplyr::mutate(
-      pts_drive = dplyr::case_when(
+      drive_pts1 = dplyr::case_when(
```

Comment:
would like to see these names be more descriptive. eg: drive_pts_rules vs drive_pts_calculated to reflect both ways of calculating them

Reply:
Good suggestion and added. Also added comments to explain the logic for choosing one over the other that came from looking row-by-row at when they disagreed and why that happened. Most disagreements were just bad drive level data